crates/bevy_ecs/src/world/mod.rs

```diff
@@ -177,10 +177,23 @@ impl World {
         WorldCell::new(self)
     }
 
+    /// Initializes a new [`Component`] type within this [`World`] and returns the
```

Comment:
```suggestion
    /// Initializes a new [`Component`] type and returns the
```

`World` method, no need to specify that.